### PR TITLE
[RU] ch5 

### PR DESCRIPTION
**Pull request by pdumin**
sections 1-4 finished

### CODE REVIEW

**Comment by pdumin** [comment]
Meanwhile chapter 5 is fully translated🤗

File: chapters/ru/chapter5/1.mdx [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Введение
+
+В [главе 3] (/course/ru/chapter3) вы поверхностно ознакомились с библиотекой 🤗 Datasets и увидели три главных шага для использования ее в процессе fine-tuning:
```
Comment (lewtun) [inline chapters/ru/chapter5/1.mdx:3]: Not sure if the `B` needs to be included in the link, but we need the space to be removed :)
```suggestion
В [главе 3](/course/ru/chapter3) вы поверхностно ознакомились с библиотекой 🤗 Datasets и увидели три главных шага для использования ее в процессе fine-tuning: 
```

**Review by lewtun — APPROVED** [review]
Thanks for translating the whole of chapter 5 @pdumin - this looks great! I've spotted one tiny issue we need to fix, but otherwise this LGTM 🔥

**Comment by lewtun** [comment]
Thanks for iterating @pdumin - looks great!

**Pull request merged**